### PR TITLE
Add Book model

### PR DESCRIPTION
**Pull request by Martum**
### Summary
- Add Book model with validations

### Trello card
https://trello.com/c/KujMuJMS/21-crear-modelo-de-book

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Book < ApplicationRecord
+
+  # Validations
```
Comment (francoprud) [inline app/models/book.rb]: I prefer not to have this comment

File: app/models/book.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  validates :genre, presence: true, allow_blank: false
```
Comment (francoprud) [inline app/models/book.rb]: I prefer to merge all the attributes inside a one liner, but it's not defined inside the company, are just different styles..

File: db/migrate/20170306185825_create_books.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateBooks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :books do |t|
```
Comment (francoprud) [inline db/migrate/20170306185825_create_books.rb]: Add timestamps to the table

File: app/models/book.rb (hunk 2 above)
Comment (Martum) [inline app/models/book.rb]: No way! Each attribute validation must has its own line! :laughing:

**Pull request merged**